tools/robustio_windows.go

```diff
@@ -0,0 +1,43 @@
+// +build windows
+
+package tools
+
+import (
+	"os"
+	"syscall"
+
+	"github.com/avast/retry-go"
+)
+
+const (
+	// This is private in [go]/src/internal/syscall/windows/syscall_windows.go :(
+	ERROR_SHARING_VIOLATION syscall.Errno = 32
+)
+
+// isEphemeralError returns true if err may be resolved by waiting.
+func isEphemeralError(err error) bool {
+	return err == ERROR_SHARING_VIOLATION
```

Comment:
Bug: I need to use `errors.Is` instead of `==` because `os.foo()` wraps underlying error.